modules/misunderstood/src/views/full/SideList.tsx

```diff
@@ -0,0 +1,67 @@
+import { Button, Icon, Intent, Tab, Tabs } from '@blueprintjs/core'
+import clsx from 'clsx'
+import React from 'react'
+
+import { FLAGGED_MESSAGE_STATUS } from '../../types'
+
+import style from './style.scss'
+import { REASONS, STATUSES } from './util'
+
+const SideList = ({
+  eventCounts,
+  selectedStatus,
+  events,
+  selectedEventIndex,
+  onSelectedStatusChange,
+  onSelectedEventChange,
+  applyAllPending
+}) => {
+  if (!eventCounts || selectedStatus == null) {
+    return null
+  }
+
+  return (
+    <div className={style.sideList}>
+      <Tabs
+        className={style.contentFixed}
+        id="StatusSelect"
+        onChange={onSelectedStatusChange}
+        selectedTabId={selectedStatus}
+      >
+        {STATUSES.map(({ key, label }) => (
+          <Tab id={key} key={key} title={`${label} (${eventCounts[key] || 0})`} />
+        ))}
+      </Tabs>
+
+      {selectedStatus === FLAGGED_MESSAGE_STATUS.pending && events && events.length > 0 && (
+        <Button onClick={applyAllPending} intent={Intent.WARNING} icon="export" className="bp3-fill">
+          Apply all pending
+        </Button>
+      )}
```

Comment:
You can set fill={true} instead of using the bp3 class

Reply:
Ah, missed it in the props list! Thanks.